io.github.zzzgydi.clash_verge.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.zzzgydi.clash_verge</id>
+  
+  <name>Clash Verge</name>
+  <summary>A Clash GUI based on tauri.</summary>
+  <developer_name>GyDi</developer_name>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <url type="homepage">https://github.com/zzzgydi/clash-verge</url>
+  <url type="bugtracker">https://github.com/zzzgydi/clash-verge/issues</url>
+  
+  <description>
+    <p>
+      A Clash GUI based on tauri.
```

Comment:
it still doesn't say what Clash is. This is a poor description. On the other hand no user care about "Tauri"

Reply:
I've added this description